ui/apps/platform/src/Containers/Compliance/Entity/Cluster.js

```diff
@@ -18,12 +19,31 @@ import isGQLLoading from 'utils/gqlLoading';
 
 import Header from './Header';
 import ResourceTabs from './ResourceTabs';
+import { isComplianceRouteEnabled } from '../complianceRBAC';
+
+function getResourceTabs({
+    isComplianceRouteEnabledForDeployments,
+    isComplianceRouteEnabledForNamespaces,
+    isComplianceRouteEnabledForNodes,
+}) {
+    const resourceTabs = [entityTypes.CONTROL];
+    if (isComplianceRouteEnabledForNamespaces) {
+        resourceTabs.push(entityTypes.NAMESPACE);
+    }
+    if (isComplianceRouteEnabledForNodes) {
+        resourceTabs.push(entityTypes.NODE);
+    }
+    if (isComplianceRouteEnabledForDeployments) {
+        resourceTabs.push(entityTypes.DEPLOYMENT);
+    }
```

Comment:
Mental slow cooker thinking ahead for RBAC at scale for Configuration Management (for which this is practice) and after updating opname object above, when I refactor the route helpers for reuse, I will also replace initial route keys like `'compliance/clusters'` with the internal keys like `'CLUSTERS'` to allow simple array 'filter` method.